tests/acceptance/features/apiContract/copy.feature

```diff
@@ -19,7 +19,7 @@ Feature: Copy test
     When user "Alice" copies file "testfile.txt" from space "new-space" to "/new/testfile.txt" inside space "new-space" using the WebDAV API
     Then the HTTP status code should be "201"
     And the following headers should match these regular expressions
-      | Oc-Fileid                   | /^[a-f0-9!\$\-]{110}$/ |
-      | Access-Control-Allow-Origin | /^[*]{1}$/             |
-      | X-Request-Id                | /^[a-f0-9!\-]{36}$/    |
+      | Oc-Fileid                   | /^[a-f0-9!\$\-]{110}$/    |
+      | Access-Control-Allow-Origin | /^[*]{1}$/                |
+      | X-Request-Id                | /^[^\/]+\/[^:]+:\d+-\d+$/ |
```

Comment:
if we want to be more strict to the scenario line pattern. Also, it matches likes like `apiContract/propfind.feature:20` if there will be
```suggestion
      | X-Request-Id                | /^[a-zA-Z]+\/[a-zA-Z]+.feature:\d+(-\d+)?$/ |
```

Reply:
UPDATED:
```diff
-^[a-zA-Z]+\/[a-zA-Z]+.feature:\d+(-\d+)?$
+^[a-zA-Z]+\/[a-zA-Z]+\.feature:\d+(-\d+)?$
```